### ISSUE TITLE
Unused import warning with Scala 2.11

### ISSUE THREAD

**Issue by limansky** [zamblauskas/scala-csv-parser#17]
Hi!

I've found that compiling with `-Ywarn-unused-import` produces really weird warning:

```
[warn] /home/limansky/projects/myproject/src/main/scala/org/mypackage/MyClass.scala:1:0: Unused import
[warn] package org.mypackage
[warn] ^
```

It looks like the issue reproduces only with Scala 2.11. I've googled little bit, it seems it related to macros, and play-json faced with same issue (they were able to solve it).

**Comment by zamblauskas** [comment]
Hi, 
After a bit of googling turns out quite a few libraries are hit by this bug, but unclear how this could be solved.

Will dig into this sometime later.

**Comment by zamblauskas** [comment]
Fixed on 0.11.6
Turns out macro code was actually adding multiple identical imports.
